_posts/2021-11-22-Technology-and-abstraction.md

```diff
@@ -9,22 +9,21 @@ modified: 2020-11-04T16:03:49-04:00
+Technology evangelists see economics as a prime target for AI because it’s a clear example of a complex system, one where thousands of variables interact with each other, and so assume that AI will soon replace economists. However this is likely an oversimplification, and it seems unlikely that we can simply create a model solely driven by AI which will make meaningful forecasts. Instead I believe that AI needs to be deployed to model certain aspects of the economy where it is clearly the best tool for the job. These are parts of an economic model where outcomes are likely influenced by a huge number of factors and conventional cause and effect models simply don’t work. Examples could be in agent behaviour-based simulations, GDP prediction based on a country's input factors (education level, stability, geography, etc.) and exchange rate prediction.
 
-There must be an acceptance that the best way to model certain parts of the economy will require abstracting beyond the point of easily understanding, but that this does not somehow detract from Economics as a discipline. Creating accurate AI driven models for specific parts of the economy facilitates the building of superior global economic models, which in turns allows higher level overarching questions to be tested and then proved or disproved via the use of simulation and predictions. 
-
-Arguably AI is an essential tool to building global economic models, and it may not even be possible to build a truly useful model without the use of AI. However it is still just one aspect in an overall approach to building a global economic model and is unlikely to be a silver bullet in it’s own right.   
+There must be an acceptance that the best way to model certain parts of the economy will require non-abstracted analysis beyond the point of easy understanding, but that this does not somehow detract from economics as a discipline. Creating accurate AI driven models for specific parts of the economy facilitates the building of superior global economic models, which in turns allows higher level overarching questions to be tested and then proved or disproved via the use of simulation and predictions. 
 
+Arguably AI is an essential tool to building global economic models, and it may not even be possible to build a truly useful model without the use of AI. However it is still just one aspect in an overall approach to building a global economic model and is unlikely to be a silver bullet in it’s own right.
```

Comment:
Since AI is a big part of your project I would be cognizant of the specific language you're using. Machine Learning is not necessarily a black box, often times you end up with a computational graph or decision tree that reads like a flow chart with different thresholds. I would talk about it as deep learning and the neural networks that you were experimenting with. You can talk about how these are more suited to such a complex system because they can utilize incremental learning as opposed to simply static data, they don't rely on manual feature extraction that carries bias in its selection, and that they can uncover correlations that previous domain experts were not explicitly looking for.